README.md

```diff
@@ -68,7 +68,7 @@ Read the [User Guide](./docs/USER_GUIDE.md) for detailed documentation.
 
     ```fish
     begin
-      set -x; cd "(mktemp -d)" &&
+      set -x; set temp_dir (mktemp -d); cd "$temp_dir" &&
```

Comment:
Thanks for providing this fix!

Can you please unset the variable with `set -e tempdir` when it is no longer needed?

Reply:
yes of course 👍